main.py

```diff
@@ -313,6 +315,100 @@ async def get_running_experiment() -> Optional[int]:
     return None
 
 
+def organize_result_directory(result_dir_path: str, rid: str) -> bool:
+    """Organizes the result directory.
+    
+    It performs the following:
+        1. Read the metadata file from the RID directory and remove it.
+        2. Move the modified experiment to the RID directory.
+        3. Find the h5 result file and copy it to the RID directory.
+        4. Dump the metadata on the copied result file.
+
+    Args:
+        result_dir_path: The full path of the result directory.
+        rid: The RID string.
+
+    Returns:
+        True if the h5 result file exists.
+        Otherwise, stop organizing the result directory and return False. 
+    """
+    rid_dir_path = posixpath.join(result_dir_path, f"{rid}/")
+    # read and remove the metadata
+    metadata_path = posixpath.join(rid_dir_path, "metadata.json")
+    with open(metadata_path, encoding="utf-8") as metadata_file:
+        metadata = json.load(metadata_file)
```

Comment:
I think we should handle the case where the metadata file does not exist.

Reply:
I see. I will handle the exception.